README.md

```diff
@@ -0,0 +1,30 @@
+# Connect-4 v1.0
+
+### by Leo Chan, (c)1997
+
+![screenshot](screenshot.png)
+
+### Source information
```

Comment:
I'd removee everthing in "Source information". It's either outdated or low on real information...